server/src-rsr/migrations/37_to_36.sql

```diff
@@ -0,0 +1,41 @@
+
+CREATE OR REPLACE FUNCTION
+  hdb_catalog.insert_event_log(schema_name text, table_name text, trigger_name text, op text, row_data json)
+  RETURNS text AS $$
+  DECLARE
+    id text;
+    payload json;
+    session_variables json;
+    server_version_num int;
+    trace_context json;
+  BEGIN
+    id := gen_random_uuid();
+    server_version_num := current_setting('server_version_num');
+    IF server_version_num >= 90600 THEN
+      session_variables := current_setting('hasura.user', 't');
+      trace_context := current_setting('hasura.tracecontext');
```

Comment:
You'll need to [explicitly pass `t`](https://www.postgresql.org/docs/current/functions-admin.html#FUNCTIONS-ADMIN-SET-TABLE) to the second argument, `missing_ok`.